tests/validation/tests/v3_api/common.py

```diff
@@ -639,24 +686,59 @@ def validate_dns_record(pod, record, expected):
 
 
 def validate_dns_entry(pod, host, expected):
+    print("Expected target:", expected)
+
     # requires pod with `dig` available - TEST_IMAGE
     cmd = 'ping -c 1 -W 1 {0}'.format(host)
-    ping_output = kubectl_pod_exec(pod, cmd)
+    if is_windows_os:
+        cmd = 'powershell -NoLogo -NonInteractive -Command "& {{ ping -w 1 -n 1 {0} }}"'.format(host)
```

Comment:
I think `cmd` has `ping` command so we don't need to use powershell here.

Reply:
Updated